src/main/resources/resources/logisim/strings/std/std_pl.properties

```diff
@@ -774,6 +774,7 @@ TTL7414 = 7414: falownik sześciokątny (schmitt-trigger)
 # ==> TTL74161 =
 # ==> TTL74163 =
 TTL74165 = 74165: 8-bitowy rejestr przesunięcia równoległego do szeregowego
+TTL74166 = 74166: 8-bitowy rejestr przesunięcia równoległego do szeregowego z wyczyść
```

Comment:
If possible, please be more precise to state that the clear is asynchronous.

@MarcinOrlowski: Could you maybe help?

Reply:
done